src/components/SignUp/SignUp.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import './SignUp.css'
+
+import Form from 'react-bootstrap/Form'
+import Button from 'react-bootstrap/Button'
+import Jumbotron from 'react-bootstrap/Jumbotron'
+import Container from 'react-bootstrap/Container'
+
+function SignUp () {
+  return (
+    <div className='SignUp'>
```

Comment:
Be careful when using global classes. CSS class names are global and can easily cause problems. I recommend that you configure [**PostCSS Modules**](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/) in your repo. It'll allow you to use any class name without worrying about global conflicts.